apps/next/src/pages/Bridge/components/BridgeTransactionForm/components/BridgeControls/components/TargetSelector.tsx

```diff
@@ -0,0 +1,34 @@
+import { Card } from '@/components/Card';
+import { BridgeTokenCard } from '@/pages/Bridge/components/BridgeTokenCard';
+import { useBridgeState } from '@/pages/Bridge/contexts';
+import { noop } from 'lodash';
+import { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import * as Styled from '../../../../Styled';
+import { NetworkSelect } from './NetworkSelect';
+
+export const TargetSelector: FC = () => {
+  const { t } = useTranslation();
+
+  const { targetNetworkId, targetNetworkIds, targetToken, amountAfterFee } =
+    useBridgeState();
+
+  return (
+    <Card noPadding>
+      <NetworkSelect
+        label={t('To')}
+        chains={targetNetworkIds}
+        selected={targetNetworkId}
+        onSelect={noop}
+        disabled
```

Comment:
This may not necessarily be true, as far as I'm aware.

I've reached out to @shelbyleblanc who's a QA for the AvaCloud team and could potentially help us test bridging tokens that can be transferred from C-Chain to multiple other chains.